integration/spark/src/main/scala/org/apache/spark/sql/CarbonSqlParser.scala

```diff
@@ -926,14 +926,14 @@ class CarbonSqlParser()
       (INTO ~> TABLE ~> (ident <~ ".").? ~ ident) ~
       (OPTIONS ~> "(" ~> repsep(loadOptions, ",") <~ ")").? <~ opt(";") ^^ {
         case filePath ~ isOverwrite ~ table ~ partionDataOptions =>
-          val (schema, tablename) = table match {
+          val (databaseNameOp, tableName) = table match {
```

Comment:
1. Why add "Op", because this is option[], so all places using the below : 
case class LoadTable(
    databaseNameOp: Option[String],
    tableName: String,
2.Using options.dbName to get databaseName.